decidim-system/app/models/decidim/system/admin.rb

```diff
@@ -4,16 +4,13 @@ module Decidim
   module System
     # Admins are the users in charge of managing a Decidim installation.
     class Admin < ApplicationRecord
-      devise :database_authenticatable, :rememberable, :validatable, :timeoutable
+      devise :database_authenticatable, :validatable, :timeoutable
+      devise :rememberable if Decidim.enable_remember_me
```

Comment:
I think actually we don't need the `rememberable` option for system administrators.

It's not something that is used very often and it is also a potential security issue if the system admin has chosen to be remembered.